WordPress/src/main/java/org/wordpress/android/ui/posts/PostSettingsTagsActivity.java

```diff
@@ -173,6 +175,7 @@ private void onTagSelected(@NonNull String selectedTag){
             updatedText = text.substring(0, endIndex + 1) + selectedTag;
         }
         updatedText += ",";
+        updatedText = StringEscapeUtils.unescapeHtml4(updatedText);
```

Comment:
I don't think we need this, since we already unescape the tags in the list and `selectedTag` will always be unescaped. It doesn't hurt us too much though, so, not really a blocker.